src/src/CustomBuild/StorageLayout.h

```diff
@@ -138,11 +147,14 @@
   #   define DAT_OFFSET_TASKS                 4096 // each task = 2k, (1024 basic + 1024 bytes custom), 12 max
   #  endif // ifndef DAT_OFFSET_TASKS
   #  ifndef DAT_OFFSET_CONTROLLER
-  #   define DAT_OFFSET_CONTROLLER           28672 // each controller = 1k, 4 max
+  #   define DAT_OFFSET_CONTROLLER           28672 // each controller = 1k, 3 max
   #  endif // ifndef DAT_OFFSET_CONTROLLER
   #  ifndef DAT_OFFSET_CUSTOM_CONTROLLER
   #   define DAT_OFFSET_CUSTOM_CONTROLLER    32768 // each custom controller config = 1k, 4 max.
```

Comment:
This would also be `3` max now, I assume? (there are some more instances of this kind of comment, below)

Reply:
This max was referring to the available space.
But we already had it set to 3 in each build.

I can also move it  (on ESP8266) to the end of where we keep the Settings Struct, as it has enough room on the ESP8266.